docs/contribute/feedback.md

```diff
@@ -1,4 +1,4 @@
-# How to provide feedback for for ROCm documentation
+# How to provide feedback for ROCm documentation
 
 There are four standard ways to provide feedback for this repository.
 
```

Comment:
@samjwu What are the four ways to provide standard feedback? Is the feedback just for this repository or for ROCm documentation?

Reply:
They are listed in the file: https://github.com/samjwu/ROCm/blob/75b6a6cf56ea24dc3572cc60172bd13e574e05b8/docs/contribute/feedback.md

Feedback can apply to both the repo and docs.